core/lib/utils/string_convert.dart

```diff
@@ -0,0 +1,10 @@
+class StringConvert {
+ static String? writeEmptyToNull(String text) {
+    if (text.isEmpty) return null;
+    return text;
+  }
+
+ static String writeNullToEmpty(String? text) {
+    return text ?? '';
+  }
```

Comment:
what? @@

Reply:
When you use copyWith function, never can set a parameter to null. But in some cases, we need some parameters that must be null to match the condition. I create it for some cases that.